Packs/GitHub/Integrations/GitHub/README.md

```diff
@@ -2848,3 +2848,26 @@ Deletes a comment.
 #### Context Output
 
 There is no context output for this command.
+
+### GitHub-add-assignee
+***
+Adds up to 10 assignees to an issue. Users already assigned to an issue are not replaced.
+
+
+#### Base Command
+
+`GitHub-add-assignee`
+#### Input
+
+| **Argument Name** | **Description**                                          | **Required** |
+|-------------------|----------------------------------------------------------|--------------|
+| assignee          | Users to assign, can be a list of users | Required     | 
+| PR                |The number of PR/Issue to assign users to | Required     | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Assignees.assignees | String | assignees to the issue | 
+| GitHub.Assignees.assignees.login | String | login of assigned user | 
```

Comment:
remove

Reply:
why remove the outputs? my logic was that after assignment - the response is big and most of that not relevant.
having assignees and the username, is how I check in the test playbook, and also can assist the user of that command to verify if user was assigned successfully.

you think it's not needed?